docs/api/qiskit/0.19/circuit.mdx

```diff
@@ -111,7 +111,7 @@ From the plot, it is easy to see that this circuit has 12 qubits, and a collecti
 qc.width()
 ```
 
-```python
+```none
```

Comment:
I thought this was a bug, but it's explicitly defined in the artifact

```html
<div class="output text_plain highlight-none notranslate"><div class="highlight"><pre><span></span>12
```

Reply:
I'm gonna normalize this to nothing

Reply:
Sg. Bump on normalizing ipython3 to python (comment above)